app/views/shared/navigation/_complementary_nav.html.erb

```diff
@@ -0,0 +1,61 @@
+<ul class="navbar-nav navbar-nav-scroll complementary-nav pt-0">
+  <% if @project %>
+    <li class="nav-item">
+      <%= render "layouts/hera/navbar/search" %>
+    </li>
+  <% end %>
+  <% if defined?(Dradis::Pro) %>
+    <% if (can?(:read, Project) || can?(:read, Team)) && @project.blank? %>
+      <%= content_tag :li, class: class_names('nav-item', active: controller_path == 'trash') do %>
+        <%= link_to main_app.trash_index_path, class: 'nav-link' do %>
```

Comment:
why is `main_app` needed here?

Reply:
Ditto later in this file. Is there a reason we need `main_app`?

Reply:
This was carried over from Tylium and yes IIRC it's for integrations